jenkins-vars.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+# jenkins variables
+project_name: docker-kali-linux
+external_type: os
+release_type: stable
+release_tag: latest
+ls_branch: master
+build_armhf: false
```

Comment:
Don't need this any more, all the armhf logic has been pulled out of the jenkinsfile now